sec.club/src/App.js

```diff
@@ -15,12 +15,30 @@ function App() {
     }
   })
   return (
-    <ThemeProvider theme={theme}>
-      <div className="App">
-        <Header></Header>
-      </div>
-    </ThemeProvider>
+    <Router>
+      <ThemeProvider theme={theme}>
+        <div className="App">
+          <Header></Header>
+          <Switch>
+            <Route path="/slp">
+              <h1>Add SLP info page here</h1>
+            </Route>
+            <Route path="/about">
+              <h1>Add What is SEC? page here</h1>
+            </Route>
+            <Route path="/team">
+              <h1>Add team information page here</h1>
+            </Route>
+            <Route path="/contact">
+              <h1>Add contact page here</h1>
+            </Route>
+            <Route exact path="/">
+              <h1>hi this is home :)</h1>
+            </Route>
```

Comment:
I would put each of these routes in their own files

Reply:
Not sure what you mean? The Route tags have to stay within `App.js` because it relies on the parent `<Switch>` and `<Router>` tags. The temporary h1 tags that have "Add x page here" will be replaced with the actual pages that they point to.

Reply:
I'm not referring to the route tags themselves. I'm talking about the content that gets rendered in the routes themselves